test.ipynb

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
Remove the .ipynb. This is a testing file that should have been caught by the .gitignore.